runtime/config-deployer-service/ballerina/APIClient.bal

```diff
@@ -1228,6 +1260,8 @@ public class APIClient {
                         model:BackendJWT backendJwt = self.retrieveBackendJWTPolicy(apkConf, apiArtifact, backendJWTPolicy, operations, organization);
                         apiArtifact.backendJwt = backendJwt;
                         policyReferences.push(<model:BackendJwtReference>{name: backendJwt.metadata.name});
+                    } else if (policyName == "AddHeaders" || policyName == "SetHeaders" || policyName == "RemoveHeaders") {
```

Comment:
Brackets are unnecessary. We can remove them.

Reply:
Fixed in https://github.com/wso2/apk/commit/3e75dbe7ea75f11d42daa95609e178756199f8e4